src/components/Filters/FilterPopup/FilterPopup.tsx

```diff
@@ -31,22 +40,50 @@ const ContentContainer = styled(Popup.Content)({
 ContentContainer.displayName = "ContentContainer";
 
 const PopupContainer = styled.div({
-  //limit the height of the popup
+  // limit the height of the popup
   maxHeight: "45vh",
   display: "flex",
   flexDirection: "column",
 });
 PopupContainer.displayName = "PopupContainer";
 
+const ButtonContainer = styled.div({
+  // evenly space the clear and save buttons
+  display: "flex",
+  justifyContent: "space-between",
+  padding: ".833em 0 0",
+});
+ButtonContainer.displayName = "ButtonContainer";
+
+const MozillaProductButton = styled.button({
+  color: "#ffffff",
+  backgroundColor: "#0060df",
+  border: "2px solid #000000",
+  borderColor: "transparent",
+  borderRadius: "4px",
+  padding: "6px 24px",
+  fontSize: "0.875rem",
+});
+MozillaProductButton.displayName = "MozillaProductButton";
+
+const MozillaNeutralButton = styled.button({
+  color: "#5e5e72",
+  backgroundColor: "transparent",
+  border: "2px solid #000000",
+  borderColor: "#cdcdd4",
+  borderRadius: "4px",
+  padding: "6px 24px",
+  fontSize: "0.875rem",
+});
+MozillaNeutralButton.displayName = "MozillaNeutralButton";
```

Comment:
Sucks that we have to do this but :shrug:

Reply:
Agreed :( I still feel icky about combining semantic UI with the rest of the library. But short of re-implementing the whole component from scratch, I think this is about as good as we're going to get for now.

And for the record, I think reimplementing this component is something worth looking into later but for the sake of finishing up the adoption of Mozilla styling, we'll just have to leave it.